src/modules/email/adapters/zoho-mail.adapter.ts

```
@@ -0,0 +1,64 @@
+    try {
+      const url = new URL(configuredLogoUrl);
+
+      if (url.protocol !== 'https:') {
+        this.logger.warn(
+          `Logo URL is not HTTPS: ${configuredLogoUrl}, using default`,
+        );
+        return defaultLogoUrl;
+      }
+
+      return configuredLogoUrl;
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)